### PR TITLE
rename ansibleapp to apb

### PR DESCRIPTION
**Pull request by jmrodri**

### CODE REVIEW

**Comment by jmrodri** [comment]
I was able to provision with dockerhub, and was able to talk to the mock registry.

**Comment by jmrodri** [comment]
In some places in the code `apb` didn't really make sense, usually in a method name. For those I favored playbookbundle or PlaybookBundle.

**Comment by jmrodri** [comment]
@eriknelson some of the changes I made to `mock_registry.go` might affect your other work so beware.

File: cmd/mock-registry/mock_registry.go [hunk 1]
```diff
@@ -7,13 +7,13 @@ import (
...
 
-	"github.com/fusor/ansible-service-broker/pkg/ansibleapp"
+	"github.com/fusor/ansible-service-broker/pkg/apb"
 	flags "github.com/jessevdk/go-flags"
 	yaml "gopkg.in/yaml.v1"
 )
 
 type Args struct {
-	AppFile string `short:"a" long:"appfile" description:"Mock Ansible Apps yaml file"`
+	AppFile string `short:"a" long:"appfile" description:"Mock Ansible Playbook Bundles yaml file"`
```
Comment (eriknelson) [inline cmd/mock-registry/mock_registry.go]: Playbook Bundle/PlaybookBundle feels reasonable in place of "App" to me.

File: cmd/mock-registry/mock_registry.go [hunk 2]
```diff
@@ -51,24 +51,24 @@ func GetAnsibleApps(w http.ResponseWriter, r *http.Request, args *Args, pApps *[
 	json.NewEncoder(w).Encode(pApps)
 }
 
-func LoadAnsibleApps(appFile string) *[]ansibleapp.Spec {
+func LoadPlaybookBundles(appFile string) *[]apb.Spec {
 	// TODO: Is this required just to unwrap the root key and get the array?
 	// Load just an array without a root key to wrap it?
 	var parsedDat struct {
-		AnsibleApps []ansibleapp.Spec
+		PlaybookBundles []apb.Spec
```
Comment (eriknelson) [inline cmd/mock-registry/mock_registry.go]: Should we get more specific with "spec" files, such as `BundleSpec`?

File: docs/aa_integration.md [hunk 3]
```diff
@@ -1,34 +1,34 @@
-# AnsibleApp Integration
+# Ansible Playbook Bundles (APB) Integration
 
-## Running AnsibleApps with the ASB
+## Running APB with the ASB
```
Comment (eriknelson) [inline docs/aa_integration.md]: Keeping track of these two acronyms when referenced closely is going to get difficult :/

**Comment by eriknelson** [comment]
@jmrodri RE: mock_registry.go changes, I don't expect `demo-broker` branch to come back into master, sort of a one off thing.

File: docs/aa_integration.md (hunk 3 above)
Comment (jmrodri) [inline docs/aa_integration.md]: I'll update the doc to be more clear. I'll make that PlaybookBundle instead.

File: cmd/mock-registry/mock_registry.go (hunk 2 above)
Comment (jmrodri) [inline cmd/mock-registry/mock_registry.go]: I like that BundleSpec would be nicer. I can make that change.

**Pull request merged**